src/com/projectkorra/projectkorra/earthbending/EarthSmash.java

```diff
@@ -731,6 +754,14 @@ public String getName() {
 		return "EarthSmash";
 	}
 
+	@Override
+	public String getInstructions() {
+		String instructions = super.getInstructions();
+		if (getConfig().getBoolean("Abilities.Earth.EarthSmash.Bounce.Enabled", false))
+			instructions += getLanguageConfig().getString("Abilities.Earth.EarthSmash.Bounce", "");
+		return instructions;
+	}
+
```

Comment:
All other abilities that have configurable features don't have separate instructions. So although it'd be better if they did, for now, we should keep to convention. So the bounce feature itself should be added to the default instructions with `\n`. If they disable the Bounce feature, they can remove it from instructions. But this is how all other abilities currently do it.